src/vorta/store/migrations.py

```diff
@@ -250,6 +251,39 @@ def run_migrations(current_schema, db_connection):
             ),
         )
 
+    if current_schema.version < 23:
+        _apply_schema_update(
+            current_schema,
+            23,
+            migrator.add_column(
+                BackupProfileModel._meta.table_name,
+                'raw_exclusions',
+                pw.CharField(default=''),
+            ),
+        )
+
+    # convert every pattern in the old exclude_patterns string to a new ExclusionModel object
+    # add a default exclusion to help the user understand how to use the new exclude GUI
+    if current_schema.version < 24:
+        for profile in BackupProfileModel:
+            previous_exclusions = profile.exclude_patterns.splitlines()
+            for pattern in previous_exclusions:
+                ExclusionModel.create(
+                    profile=profile,
+                    name=pattern,
+                    enabled=True,
+                )
```

Comment:
This can raise `peewee.IntegrityError: UNIQUE constraint failed: exclusionmodel.name`

Reply:
Done